docs/kubernetes/kctlr-multi-tenancy.rst

```diff
@@ -0,0 +1,225 @@
+.. index::
+   single: BIG-IP Controller; Kubernetes; multi-tenancy
+   single: BIG-IP Controller; Openshift; multi-tenancy
+
+.. _openshift multi-tenancy:
+
+Multi-tenancy in Kubernetes and OpenShift
+=========================================
+
+Overview
+--------
+
+BIG-IP administrative partitions allow you to isolate tenants from each other and to configure objects for specific tenants, where only authorized users can access them. To manage multi-tenant BIG-IP systems with the |kctlr-long| and OpenShift, F5 recommends deploying one (1) instance of the :code:`k8s-bigip-ctlr` per BIG-IP partition.
+
+You can set up the |kctlr| for multi-tenancy a few different ways. Each of the example use cases described here uses namespaces (or projects, in OpenShift) to achieve multi-tenancy in the Kubernetes/OpenShift Cluster.
+
+.. note::
+
+   In each use case, the various |kctlr| instances *can* use the same Service Account/Cluster Role/Cluster Role Binding. Whether or not the Controllers *should* use the same Service Account depends on the requirements of your tenants/Cluster.
+
+How to deploy
+`````````````
+
+You can deploy any of the use cases summarized here by taking the steps below:
+
+#. Create a YAML manifest containing the desired resources.
+#. Upload the manifest to the Kubernetes/OpenShift API server.
+
+   .. code-block:: bash
+
+      kubectl create -f f5-k8s_manifest.yaml [--namespace=<namespace]   \\ Kubernetes
+      oc create -f f5-k8s_manifest.yaml [--namespace=<namespace]        \\ OpenShift
```

Comment:
In OpenShift, the namespace construct is not really used. OpenShift uses Projects as a way to provide multi-tenancy. 

When you log into an OpenShift cluster, the operator or developer must specify the project they are working in. So I think we should remove the namespace switch from the `oc` command.

Reply:
@rsalsa - Is this true across the board for OpenShift? We have other `oc` command examples that use the `--namespace` designation; should I remove those as well?